sdk/identity/identity/test/internal/node/usernamePasswordCredential.spec.ts

```diff
@@ -7,12 +7,15 @@ import Sinon from "sinon";
 import { assert } from "chai";
 import { Context } from "mocha";
 import { GetTokenOptions } from "@azure/core-auth";
-import { env, isLiveMode, Recorder } from "@azure-tools/test-recorder";
+import { env, isPlaybackMode, isLiveMode, Recorder } from "@azure-tools/test-recorder";
 import { PublicClientApplication } from "@azure/msal-node";
+import { AzureLogger, setLogLevel } from "@azure/logger";
 import { UsernamePasswordCredential } from "../../../src";
 import { MsalTestCleanup, msalNodeTestSetup } from "../../msalTestUtils";
 import { MsalNode } from "../../../src/msal/nodeFlows/msalNodeCommon";
 
+const liveClientId = "04b07795-8ddb-461a-bbee-02f9e1bf7b46"; // Live test client ID
```

Comment:
This is the same client ID that .NET uses for this test: https://github.com/Azure/azure-sdk-for-net/search?q=04b07795-8ddb-461a-bbee-02f9e1bf7b46

Reply:
Oh, this client Id is the same one we use for the DeviceCodeCredential! woops. I’ll clarify here.